packages/generators/app/src/utils/usage.ts

```diff
@@ -39,10 +39,10 @@ export function captureStderr(name: string, error: unknown) {
       .trim()
       .split('\n')
       .forEach((line) => {
-        sentry.addBreadcrumb({
+        Sentry.addBreadcrumb({
           category: 'stderr',
           message: line,
-          level: Severity.Error,
+          level: Sentry.Severity.Error,
```

Comment:
We prefer string literals here instead of the enum: https://github.com/getsentry/sentry-javascript/blob/df7bb254d3f7b1c456a5949efcc123606838aa77/MIGRATION.md#L424-L428